### DISCUSSION TITLE
Expose Switch Network button without wallet connected

### DISCUSSION THREAD

**Discussion by oxSaturn** [rainbow-me/rainbowkit#1671] [Ideas]
Some similar requests before:

1. https://github.com/rainbow-me/rainbowkit/discussions/381#discussioncomment-2983189
2. https://github.com/rainbow-me/rainbowkit/pull/1601#issuecomment-1824956017

Here's why I think it's better UX to show it without wallet connected.

1. users can know which chains are supported without wallet connected, take uniswap for example,
![CleanShot-JGgdbuy0@2x](https://github.com/rainbow-me/rainbowkit/assets/126733611/c18bd9ec-2392-46a8-86f5-5a41bf151bff)

    As you can see, I can tell it's deployed on which chains without having a wallet installed and connected, while with rainbowkit, it's not possible.

3. users can see the current connected chain set by the `initialChain`, currently there's no way to tell which chain is it without connected.

**Comment by oxSaturn** [comment]
cc @0xartcro @hzhu

**Comment by art-eth** [comment]
Here is another discussion about that https://github.com/rainbow-me/rainbowkit/discussions/1668
I have created one a few days ago as well ;)

**Comment by oxSaturn** [comment]
Awesome, let me close this one to favour yours.

**Comment by KulzeinepHub** [comment]
It was very interesting

**Comment by Avkubs** [comment]
Maybe that's right
